srcpkgs/libnvme/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'libnvme'
 pkgname=libnvme
-version=1.1
-revision=2
+version=1.2
+revision=1
 build_style=meson
 configure_args="-Ddocs=man"
 hostmakedepends="pkg-config swig"
-makedepends="libuuid-devel json-c-devel openssl-devel python3-devel"
+makedepends="json-c-devel openssl-devel python3-devel"
 checkdepends="perl"
 short_desc="C Library for NVM Express on Linux"
 maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/linux-nvme/libnvme"
 distfiles="https://github.com/linux-nvme/libnvme/archive/v${version}.tar.gz"
-checksum=d000181356513733671409c747c88826155b60f45f76e7289d6bbf27a8f851b0
+checksum=2697c86e9d8597fcd264f8bf8af57a529346efa4f611ca2be855e3aa89b9eb2c
 
 libnvme-devel_package() {
 	depends="${sourcepkg}>=${version}_${revision}"
```

Comment:
```suggestion
	depends="${sourcepkg}>=${version}_${revision} json-c-devel openssl-devel"
```
and remove the `depends+=` line, it's not necessary to have a comment about this because every -devel package works the same way.